pom.xml

```diff
@@ -327,6 +327,134 @@ limitations under the License.
     </profile>
 
     <profile>
+      <id>hbase-1.0.0</id>
+      <activation>
+        <property>
+          <name>hadoop.profile</name>
```

Comment:
Same comment : Why to you use hadoop.profile to set with hbase-1.0.0 and not hbase.profile ?